validation/schema.py

```diff
@@ -1,37 +1,56 @@
-from marshmallow import Schema, fields, validate
-
-class LinksetSchema(Schema):
-    orcid = fields.Str(required=True)
-    twitter = fields.Str()
-    github = fields.Str()
-    website = fields.URL(schemes=['https'])
-
-class InstitutionSchema(Schema):
-    institution = fields.Str(required=True)
-    website = fields.URL(required=True, schemes=['https'])
-
-class CuratorSchema(Schema):
-    name = fields.Str(required=True)
-    title = fields.Str(required=True)
-    affiliation = fields.Nested(InstitutionSchema(),required=True)
-    links = fields.Nested(LinksetSchema(), required=True)
-
-class PluginSchema(Schema):
-    name = fields.Str(required=True)
-    comment = fields.Str(validate=validate.Length(max=300))
-
-class CollectionSchema(Schema):
-    title = fields.Str(required=True,validate=validate.Length(max=500))
-    cover_image = fields.Str(required=True)
-    summary = fields.Str(required=True)
-    description = fields.Str(required=True)
-    plugins = fields.List(
-        fields.Nested(PluginSchema()),
-        required=True
-        )
-    updated_date = fields.Date(required=True)
-    curator = fields.Nested(CuratorSchema(), required=True)
-    visibility = fields.Str(
-        validate=validate.OneOf(('public','hidden','disabled')),
-        load_default='public',
-    )
+import datetime
+from typing import List, Optional
+from typing_extensions import Literal
```

Comment:
Depending on the python version we will use this could become

`from typing import List, Literal, Optional` and we would not need to rely on `typing_extensions`

Reply:
@goanpeca would that need 3.8?

the workflow uses 3.10: https://github.com/chanzuckerberg/napari-hub-collections/blob/main/.github/workflows/validate-collections.yml#L25

@klai95 what does the napari hub backend use?

Reply:
Yep, anything above 3.8 is fair game, but I was in compatibility mode when I wrote that 😆

Reply:
😆 

I'll defer to @klai95 but I don't see a strong case to maintain backward compatibility here.

Reply:
Fixed! need to rerun the CI (Approve) @neuromusic